modules/zenith-public/sql/resaleItemChanges.sql

```diff
@@ -0,0 +1,9 @@
+-----------------------------------
+-- Item Resale Price Adjustment
+-- Updates every base sell price of items in the game to be 95% of its original value,
+-- with a minimum value of 1 to prevent extremely low prices
+-- Public module for ZenithXI
+-----------------------------------
+
+UPDATE item_basic
+SET BaseSell = CEILING(BaseSell * 0.95); -- Reduce all item base sell prices by 5%
```

Comment:
```suggestion
SET BaseSell = CEILING(BaseSell * 0.90); -- Reduce all item base sell prices by 10%
```

Lets start this at 10% please. We can adjust it back to 5% later if necessary.

Reply:
done and amended the commit